tests/test_dwavesampler.py

```diff
@@ -203,3 +206,53 @@ def __init__(self):
                 pass
 
         DWaveSampler.validate_anneal_schedule(MockScheduleSampler(), [(0, 1), (55.0, 0.45), (155.0, 0.45), (210.0, 1)])
+
+
+class TestDWaveFailoverSampler(unittest.TestCase):
+    @mock.patch('dwave.system.samplers.dwave_sampler.Client')
+    def test_failover_offline(self, MockClient):
+        if sys.version_info.major <= 2 or sys.version_info.minor < 6:
+            raise unittest.SkipTest("need mock features only available in 3.6+")
```

Comment:
Isn't there a workaround for older Pythons? The test seems straightforward enough...

Reply:
It's the `return_value` kwarg in  https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.reset_mock. And yes, there is definitely a workaround. However, given that we're deprecating 2.7 and 3.4, this is pure python code and it's for one test I think it's simpler/cleaner to just do the simpler thing.